components/home/sections/newsletter.js

```diff
@@ -1,39 +1,58 @@
 "use client";
-
 import { useState } from "react";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
 
 export default function Newsletter() {
   const [email, setEmail] = useState("");
 
-  function HandleSubmit(e) {
-    window.location.href = `mailto:pycon-uganda+subscribe@googlegroups.com?subject=Subscribe%20to%20PyCon%20Uganda%20Newsletter`;
-  }
-
+  // function HandleSubmit(e) {
```

Comment:
```suggestion
  // function HandleSubmit(e) {
```

The dead code could be removed...

Reply:
I've removed the dead code